code/modules/urist/items/uristweapons.dm

```diff
@@ -197,3 +197,81 @@ Please keep it tidy, by which I mean put comments describing the item before the
 	throwforce = 20
 	w_class = 4.0
 	slot_flags = SLOT_BELT|SLOT_BACK
+
```

Comment:
Great work. Make sure you also add the item's name in // (brackets) as a comment version above it. It's more organized and tidy if you do so. E.g, // Crutch  (code would go below this)